web/styles/subscriptions.css

```diff
@@ -704,6 +704,10 @@ h4.user_group_setting_subsection_title {
                 45px (.subscriptions-header) +  44px (.display-type) + 60px (.modal-footer)
             */
             height: calc(95vh - 149px);
+            /*
+                1000px (max height on modal) - 149px (.subscriptions-header, .display-type, .modal-footer)
+            */
+            max-height: 851px;
```

Comment:
Can we make the 1000px and 149px into CSS variables (well, and maybe the 45/44/60 too!) and actaully do the math here in code rather than comments?

Reply:
Yes, let's! I'll put up a patch either this weekend or first thing Monday

Reply:
Patch is up. I went super-basic on the math, just subtracting all the values rather than grouping with parentheses (that led to more readable output from the linter, too).

Should be ready for your final review, @timabbott.